raiden-dapp/src/views/account/UDC.vue

```diff
@@ -1,133 +1,52 @@
 <template>
-  <div>
-    <v-row class="udc__description" justify="center" no-gutters>
-      <p>
-        {{ $t('udc.description') }}
-      </p>
+  <div class="udc">
+    <v-row class="udc__description" no-gutters>
+      {{ $t('udc.description') }}
     </v-row>
-    <v-row justify="center">
-      <v-col cols="10">
-        <h3 class="udc__sub-head">
-          {{ $t(mainnet ? 'udc.deposit' : 'udc.mint-deposit') }}
-        </h3>
-      </v-col>
+    <v-row class="udc__balance" no-gutters justify="center">
+      {{ $t('udc.balance') }}
+      <amount-display
+        class="udc__balance__amount"
+        inline
+        :amount="udcCapacity"
+        :token="udcToken"
+      />
     </v-row>
-    <v-row v-if="!loading && !error" justify="center" no-gutters>
-      <v-col cols="6">
-        <v-text-field
-          v-model="amount"
-          autofocus
-          type="text"
-          :suffix="serviceToken"
-          class="udc__mint-deposit--amount"
+    <v-row class="udc__actions" no-gutters justify="center">
+      <v-btn text class="udc__actions__button" @click="showUdcDeposit = true">
+        <v-img
+          width="70px"
+          height="65px"
```

Comment:
Nitpick^2:
I personally would prefer a smaller size. While the buttons appear fancy at the channel screen, here they look a bit chunky to me.

Reply:
I made them slightly smaller. :)